torch/csrc/nn/type_checks.h

```diff
@@ -19,7 +19,7 @@ inline bool check_type(PyObject* obj, at::TensorTypeId id, at::ScalarType dtype)
 
 template<typename T>
 inline T* unpack(PyObject* obj) {
-  return (T*) ((THPVariable*)obj)->cdata.data().unsafeGetTensorImpl();
+  return (T*) ((THPVariable*)obj)->cdata.unsafeGetTensorImpl();
```

Comment:
Based on discussion with Sam, this `unpack()` function is not used because we had already killed torch.legacy.nn and all the normal NN ops go through ATen bindings right now. I also tried removing the content of this function and run `test_nn.py`, and there was no test error. So it's okay to remove the `.data()` call here.

Reply:
why don't we just remove the function then?

Reply:
I will work on removing it after this PR.

Reply:
do you have an issue filed for this?

Reply:
I filed an issue at https://github.com/pytorch/pytorch/issues/20704.